### DISCUSSION TITLE
Welcome to leapp-repository Discussions!

### DISCUSSION THREAD

**Discussion by pirat89** [pirat89/leapp-repository#5] [General]
just a try... to see how it works, etc. if we are happy with that, we can start it on leapp repository as well.

## 👋 Welcome to the Leapp development discussion place

:green_heart: We expect and will be happy to see here:
  * Development stuff - how to implement actors of specific functionality, how to extend current functionality, ... - but ensure you are familiar with the upstream [documentation](https://leapp.readthedocs.io/en/latest/) first; especially with [tutorials](https://leapp.readthedocs.io/en/latest/tutorials.html)
  * Feedback on documentation - is it hard to get started? Let us know where is the problem / how we could improve it in future
  * If you think you are missing something, you are willing to implement the change and open PR, and you want to know whether we are ok with the change/feature/... before you implement it
  * If you are looking for help with investigation (read, looking for tips where to look, etc.; we are not going to do the investigation itself) of an issue
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and be open-minded. Remember that this is a community we build together 💪.

  :no_entry: This place is not support forum! Please contact your vendor or distribution provider for support.

**Comment by annamarie7** [comment]
Sounds great! We'd like Convert2RHEL to follow Leapp's example... Thanks for sharing, Petr!

**Reply by pirat89** [reply]
Currently we just discuss whether to start to use this and if so, what will be limitations for that. Currently, as we can count just with 4 people in our team, it's question whether we will use it or whether we postpone that now till the time we know we have capacity to handle another communication channel. But I really like it and I am happy that I am not alone :)
